src/main/java/com/ericsson/ei/frontend/EIRequestsController.java

```diff
@@ -80,10 +94,10 @@
     @RequestMapping(value = { "/subscriptions", "/rules/rule-check/aggregation", "/query" }, method = RequestMethod.POST)
     public ResponseEntity<String> postRequests(Model model, HttpServletRequest request) {
         String eiRequestUrl = getEIRequestURL(request);
-
         String requestBody = "";
+
         try {
-            requestBody = request.getReader().lines().collect(Collectors.joining(System.lineSeparator()));
+            requestBody = request.getReader().lines().collect(Collectors.joining(System.lineSeparator())).replaceAll("(\\r)", "");
```

Comment:
What is this? Do we have garbage character in http request?

Reply:
In Unix systems a line break is indicated with \n. In windows thay add \r\n that means the same thing. The \r must be removed for tests to pass in windows.

Reply:
If I understand this correctly, this is only issue while running as test in Windows and not when running it in sharp/target environment.
Then this replace of windows line breaks/returns should only be in tests class where you read the result.

Reply:
That is sadly not possible to do since it is a mockserver that returns weather it got the expected request or not.

Reply:
Have you tried setting setProperty lineseparator parameter from the test code?
https://stackoverflow.com/questions/22681534/java-change-system-new-line-character